tests/playwright/sanity/image-carousel.test.js

```diff
@@ -3,8 +3,9 @@ const WpAdminPage = require( '../pages/wp-admin-page.js' );
 
 test( 'Image Carousel', async ( { page }, testInfo ) => {
 	// Arrange.
-	const wpAdmin = new WpAdminPage( page, testInfo ),
-		editor = await wpAdmin.useElementorCleanPost();
+	const wpAdmin = new WpAdminPage( page, testInfo );
+
+  editor = await wpAdmin.useElementorCleanPost();
```

Comment:
shouldn't the `editor` have `const editor` and can't we remove the empty line? Compare other tests.